### PR TITLE
howtos: process: Define bug tracking workflow

### PR DESCRIPTION
**Pull request by mengdonglin**
This patch defines the life cycle of a bug, labels to assist bug
tracking, and how to report or close a bug.

I plan to add this into the contribute guideline later as a link later.

### CODE REVIEW

File: howtos/process/bug-tracking.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+How to Report a Bug
+********************
+Please
+`create a issue <https://help.github.com/articles/creating-an-issue/>`_
+and add label |label-bug|.
+
+Please provide the following information:
+
+* A clear bug title that is a concise description of the bug.
```
Comment (lgirdwood) [inline howtos/process/bug-tracking.rst]: Te bug title must also be unique and descriptive. Bad examples are "ipc timout" and "topology failed to load". Idealy the title should contain some text from the kernel , FW, or user space error message.

File: howtos/process/bug-tracking.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+* Branch name and commit hash of 3 repositories: sof (firmware),
+  linux (kernel driver) and soft (tools & topology).
+
+* Name of the topology file
+
+* Name of the platform(s) on which the bug is observed.
+
+* Kernel message and firmware logger data
+    * Please paste the relevant dmesg and logger data to the comment box.
```
Comment (lgirdwood) [inline howtos/process/bug-tracking.rst]: They should use markdown for formatting example commands, code, diffs, patches etc. Can you state this and link to markdown guide.

Comment (mengdonglin) [reply howtos/process/bug-tracking.rst]: Okay. Updated the PR.

**Comment by mengdonglin** [comment]
Still need some update. I'll submit a new PR

**Pull request closed without merging**